airbyte-integrations/connectors/source-amazon-seller-partner/acceptance-test-config.yml

```diff
@@ -3,7 +3,7 @@ tests:
   spec:
     - spec_path: "integration_tests/spec.json"
       backward_compatibility_tests_config:
```

Comment:
Remove `backward_compatibility_tests_config`.
```suggestion
```

Reply:
@monai it is already removed.
Please see:
https://github.com/airbytehq/airbyte/pull/18283/commits/9cfdd44abe76342b4c8fec81937d8c2ba0b0610a